lib/common/client/concerns/mhv_jwt_session_client.rb

```diff
@@ -139,10 +140,10 @@ def auth_body
           {
             'appId' => '103',
             'appToken' => config.app_token,
-            'subject' => session.user_id.to_s,
+            'subject' => session.icn,
```

Comment:
`to_s` no longer needed here?

Reply:
@rmtolmach that's correct. user_id is a numeric value while icn is a string that contains alphanumeric characters.